CHANGELOG.md

```diff
@@ -4,6 +4,61 @@ All notable changes to this project will be documented in this file.
 
 ## Unreleased
 
+## 0.9.0
+
+#### :boom: Breaking Change
```

Comment:
Is it possible to make these level 3 headings i.e. ### ?

Reply:
done.